src/query/functions/aggregation/quantile_test.go

```diff
@@ -117,6 +117,18 @@ func TestQuantileFnSingleNonNan(t *testing.T) {
 	test.EqualsWithNansWithDelta(t, expected, actual, math.Pow10(-5))
 }
 
+func TestQuantileNanAndEmptyArguments(t *testing.T) {
+	actual := []float64{
```

Comment:
This is a good idea, but it will not report which case(s) fail.
There is a more standard approach for such tests in go, called 'test tables`, eg.:
https://github.com/m3db/m3/blob/12dc12e118fa4b7ed9e24fa7976e4ec0d3de488f/src/dbnode/ts/downsample/counter_resets_downsampler_test.go#L31